### PR TITLE
[NEEDS REVIEW] Added a new level

### LINKED ISSUE #1023: Add a level
## Expected Behavior

Add a level with new tiles.

### PR DESCRIPTION
**Pull request by sahilsaha7773**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [ ] Tried to [squash the commits][squash].
- [x] Added Screenshot / Path to tile to Indicate Location of Change in Game

### Changes done in this Pull Request

- View my work at https://pr-1022-fossasia-labyrinth.surge.sh or https://sahilsaha7773.github.io/labyrinth/

I have added the new level and tiles in collaboration with @tantrojan (Tanmoy Ghosh)  

- Fixes #1023 

### The problem I want to solve or the facility I want to improve is/are

Add a new level with tiles.

### Screenshots
Path to level selector tile :- `RIGHT,UP,RIGHT,UP,UP,UP,RIGHT,UP,UP,UP`

anl2
anl

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

**Comment by sahilsaha7773** [comment]
As we have done this in collaboration I think not squashing the commits will be better @pythongiant

File: js/tiles.js [hunk 1]
```diff
@@ -1657,6 +1658,79 @@ const forest = {
...
+        canLeaveToTheTop: function(player) {
+            return false;
+        },
+        createImages: function() {
+            this.wallTop = this.createImage("tiles/rooms/wall/topHexagon.svg");
+            this.wallRight = this.createImage("tiles/rooms/door/rightHexagon.svg");
+            this.ground = this.createImage("tiles/rooms/floor/Desert.svg");
+        },
+    }),
+    start: Object.assign({}, OpenDoors, {
```
Comment (pythongiant) [inline js/tiles.js]: work at only one issue at one time

Comment (sahilsaha7773) [reply js/tiles.js]: I solved one issue only

**Comment by sahilsaha7773** [comment]
I found the Travis build to fail in each and every PR

**Comment by sahilsaha7773** [comment]
I think there might be some problems with the .travis.yml file

File: js/tiles.js (hunk 1 above)
Comment (pythongiant) [reply js/tiles.js]: no you didnt. I see more than one walls

Comment (sahilsaha7773) [reply js/tiles.js]: I have created a level so it will consist of more than one wall

**Comment by sahilsaha7773** [comment]
@YashKumarVerma I have added the path to the tile and screenshots are already provided :)

**Comment by abishekvashok** [comment]
Thanks

**Pull request merged**